### PR TITLE
Clarify docstring descriptions of cross-axis slope

### PR DESCRIPTION
**Pull request by chiragpachori**
The previous description for the signs of cross axis tilt mentioned slope azimuth as east for both positive and negative. Corrected it to indicate that sign of cross axis tilt is positive when axis azimuth is south and slope is towards west.

 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Pull request is complete and ready for detailed review.

This is a minor documentation error and I used the tracking.calc_cross_axis_tilt function to make sure that I an getting positive sign when slope is facing west.

### CODE REVIEW

**Comment by cwhanse** [comment]
We're all for clarifying docstrings.

The existing docstring says "**negative** cross-axis tilt if the tracker axes plane slopes **down** to the east and **positive**
cross-axis tilt if the tracker axes plane slopes **up** to the east" (emphasis mine) which isn't contradictory but perhaps is difficult to parse.

Would "**positive** cross-axis tilt if the tracker axes plane slopes **down** to the west" be more clear?

**Comment by chiragpachori** [comment]
Ohh I see! Apologies for missing that part. 

I think phrasing it as "positive cross-axis tilt if the tracker axes plane slopes **down** to the **west**" might make it more clear. As it will line up well with the other definitions throughout the documentation, where sloping down is generally used to to get the direction. For example (from the same file): 

1. slope_azimuth : float
        direction of the normal to the slope containing the tracker axes, when
        projected on the horizontal [degrees]
2. `surface_azimuth`: The azimuth of the rotated panel, determined by
          projecting the vector normal to the panel's surface to the earth's
          surface. [degrees]

In both these cases the intuitive reference is sloping down, at least from how I understand it.

**Comment by cwhanse** [comment]
> the intuitive reference is sloping down

I agree.

**Comment by cwhanse** [comment]
@chiragpachori will you be able to update this PR? If not, one of us can do it, just let us know.

File: docs/sphinx/source/whatsnew/v0.9.3.rst [hunk 1]
```diff
@@ -35,6 +35,7 @@ Testing
 
 Documentation
 ~~~~~~~~~~~~~
+* Clarified description of cross-axis slope in :py:mod:`pvlib.tracking.py` (:pull:`1530`)
```
Comment (kanderso-nrel) [inline docs/sphinx/source/whatsnew/v0.9.3.rst:38]:
```suggestion
* Clarified description of cross-axis slope in :py:mod:`pvlib.tracking` (:pull:`1530`)
```

File: docs/sphinx/source/whatsnew/v0.9.3.rst [hunk 2]
```diff
@@ -57,4 +58,5 @@ Contributors
 * Mark Campanelli (:ghuser:`markcampanelli`)
 * Adam R. Jensen (:ghuser:`AdamRJensen`)
 * Abhishek Parikh (:ghuser:`abhisheksparikh`)
-* Taos Transue (:ghuser:`reepoi`)
\ No newline at end of file
+* Taos Transue (:ghuser:`reepoi`)
+* (:ghuser:`chirapachori`)
```
Comment (kanderso-nrel) [inline docs/sphinx/source/whatsnew/v0.9.3.rst:62]:
```suggestion
* (:ghuser:`chiragpachori`)
```

**Review by kanderso-nrel — APPROVED** [review]

**Comment by kanderso-nrel** [comment]
Thanks @chiragpachori!

**Comment by chiragpachori** [comment]
@cwhanse Sorry I missed your comment. Thank you for sorting it out. @kanderso-nrel

**Pull request merged**